ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -71,7 +71,13 @@ enum CommandDeduplicationType {
 
 // See `daml-lf/spec/contract-id.rst` for more information on contract ID formats.
 message ContractIdFeatures {
-  ContractIdV1Support v1 = 1;
+  ContractIdV0Support v0 = 1;
```

Comment:
We do not really need a feature for V0. V0 should be rejected by all supported ledgers.
I think what @cocreature want is that we check that V0 is always rejected, but we can drop the tests that check V0 is supported.

Reply:
yep

Reply:
```suggestion
```

Reply:
Currently Sandbox-on-X supports v0. Should we change this?

Reply:
(I would prefer not to change this as part of this PR.)

Reply:
we should but agreed, let’s do it in a separate PR.